### PR TITLE
Test

### PR DESCRIPTION
**Pull request by cedricRobert**
- add cmake option : WITH_TEST
  This option add log for perf test

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -34,6 +34,13 @@ ELSE(WITH_ETP)
 	LIST (REMOVE_ITEM FESPP_FORMS ${FESPP_FORMSETP})
 ENDIF (WITH_ETP)
 
+IF (WITH_TEST)
```
Comment (philippeVerney) [inline src/CMakeLists.txt]: IF (NOT WITH_TEST) will allow to remove the comment

File: src/PQDataLoadManager.cxx [hunk 2]
```diff
@@ -111,20 +111,22 @@ void PQDataLoadManager::setupPipeline()
 		fesppReader = builder->createReader("sources", "Fespp", QStringList("EpcDocument"), this->Server);
 		manager->existPipe(true);
 	}
+	QStringList epcFiles =	this->ui->epcFile->filenames();//.join("*");
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: Remove comment which does not explain anything. Or explain it.

File: src/PQDataLoadManager.cxx [hunk 3]
```diff
@@ -111,20 +111,22 @@ void PQDataLoadManager::setupPipeline()
 		fesppReader = builder->createReader("sources", "Fespp", QStringList("EpcDocument"), this->Server);
 		manager->existPipe(true);
 	}
+	QStringList epcFiles =	this->ui->epcFile->filenames();//.join("*");
 
-	QString epcFiles = this->ui->epcFile->filenames().join("*");
+//	QString epcFiles = this->ui->epcFile->filenames().join("*");
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: Remove comment which does not explain anything. Or explain it.

File: src/PQDataLoadManager.cxx [hunk 4]
```diff
@@ -111,20 +111,22 @@ void PQDataLoadManager::setupPipeline()
...
-		if (fesppReader) {
+	if (!epcFiles.isEmpty() && fesppReader) 	{
+
+		for (auto it=0; it < epcFiles.length(); ++it){
 			pqActiveObjects *activeObjects = &pqActiveObjects::instance();
 			activeObjects->setActiveSource(fesppReader);
 
 			vtkSMProxy* fesppReaderProxy = fesppReader->getProxy();
-			vtkSMPropertyHelper( fesppReaderProxy, "SubFileName" ).Set(epcFiles.toStdString().c_str());
+			vtkSMPropertyHelper( fesppReaderProxy, "SubFileName" ).Set(epcFiles[it].toStdString().c_str());
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: Commonly "it" means "iterator".
Here this is an index not an interator.

File: src/PQEtpPanel.cxx [hunk 5]
```diff
@@ -60,23 +60,30 @@ void PQEtpPanel::handleButtonRefresh()
 {
 	VtkEtpDocument etp_document(ipAddress, port, VtkEpcCommon::TreeView);
 
+	// Wait for etp connection
+	cout << "av attente getClientSession()\n";
```
Comment (philippeVerney) [inline src/PQEtpPanel.cxx]: No output in french

File: src/PQEtpPanel.cxx [hunk 6]
```diff
@@ -60,23 +60,30 @@ void PQEtpPanel::handleButtonRefresh()
...
+	cout << "av attente getClientSession()->isEtpSessionClosed() OK\n";
+
 	QIcon icon;
 	icon.addFile(QString::fromUtf8(":green_status.png"), QSize(), QIcon::Normal, QIcon::Off);
 	EtpStatus_Button->setIcon(icon);
 
-	// Wait for etp connection
-	while (etp_document.getClientSession() == nullptr) {
+	etp_document.createTree();
+	if (EtpSendButton->text() == "Create TreeView"){
```
Comment (philippeVerney) [inline src/PQEtpPanel.cxx]: Only class (not variable) should start with an upper case.

File: src/PQSelectionPanel.cxx [hunk 7]
```diff
@@ -99,52 +99,59 @@ knowledge of the CeCILL license and that you accept its terms.
...
+	return panel;
+}
 #endif
 
-	pqPropertiesPanel* getpqPropertiesPanel() {
-		pqPropertiesPanel *panel = 0;
-		foreach(QWidget *widget, qApp->topLevelWidgets()) {
-			panel = widget->findChild<pqPropertiesPanel *>();
+pqPropertiesPanel* getpqPropertiesPanel() {
+	pqPropertiesPanel *panel = 0;
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: nullptr instead of zero

File: src/PQSelectionPanel.cxx [hunk 8]
```diff
@@ -99,52 +99,59 @@ knowledge of the CeCILL license and that you accept its terms.
...
+	foreach(QWidget *widget, qApp->topLevelWidgets())
+	{
+		panel = widget->findChild<pqPropertiesPanel *>();
 
-			if (panel) {
-				break;
-			}
+		if (panel) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: nullptr check?

File: src/PQSelectionPanel.cxx [hunk 9]
```diff
@@ -210,56 +228,61 @@ void PQSelectionPanel::constructor()
...
 //----------------------------------------------------------------------------
-void PQSelectionPanel::onItemCheckedUnchecked(QTreeWidgetItem * item, int column)
-{
+void PQSelectionPanel::onItemCheckedUnchecked(QTreeWidgetItem * item,
+		int column) {
 	auto uuid = itemUuid[item];
 
-	if (!(uuid == ""))	{
-		if (item->checkState(0) == Qt::Checked)	{
+	if (!(uuid == "")) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: use "(not)empty" method instead of equality with ""

File: src/PQSelectionPanel.cxx [hunk 10]
```diff
@@ -273,62 +296,65 @@ void PQSelectionPanel::onItemCheckedUnchecked(QTreeWidgetItem * item, int column
...
-		radioButton_to_uuid.clear();
+	pcksave.clear();
+	uuidParent_to_groupButton.clear();
+	radioButton_to_uuid.clear();
 
-		if (vtkEpcDocumentSet){
-			delete vtkEpcDocumentSet;
-			vtkEpcDocumentSet = new VtkEpcDocumentSet(0, 0, VtkEpcCommon::TreeView);
-		}
+	if (vtkEpcDocumentSet) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: nullptr check. This is not a booelan

File: src/PQSelectionPanel.cxx [hunk 11]
```diff
@@ -617,32 +647,32 @@ void PQSelectionPanel::addTreeProperty(QTreeWidgetItem *parent, const std::strin
...
+	}
 
 	uuidParent_to_groupButton[itemUuid[parent]] = buttonGroup;
 }
 
 //****************************************************************************
-void PQSelectionPanel::deleteUUID(QTreeWidgetItem *item)
-{
+void PQSelectionPanel::deleteUUID(QTreeWidgetItem *item) {
 	for (int o = 0; item->childCount(); o++) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: Should it no be unsigned int?

File: src/PQSelectionPanel.cxx [hunk 12]
```diff
@@ -617,32 +647,32 @@ void PQSelectionPanel::addTreeProperty(QTreeWidgetItem *parent, const std::strin
...
 }
 
 //----------------------------------------------------------------------------
-void PQSelectionPanel::loadUuid(const std::string & uuid)
-{
+void PQSelectionPanel::loadUuid(const std::string & uuid) {
 	auto pipe_name = searchSource(uuid);
 	pqPipelineSource * source = findPipelineSource(pipe_name.c_str());
-	if (source)	{
+	if (source) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: nullptr

File: src/PQSelectionPanel.cxx [hunk 13]
```diff
@@ -667,11 +698,10 @@ void PQSelectionPanel::loadUuid(const std::string & uuid)
 }
 
 //----------------------------------------------------------------------------
-void PQSelectionPanel::removeUuid(const std::string & uuid)
-{
+void PQSelectionPanel::removeUuid(const std::string & uuid) {
 	auto sourceName = searchSource(uuid);
 	pqPipelineSource * source = findPipelineSource(sourceName.c_str());
-	if (source)	{
+	if (source) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: nullptr

File: src/PQSelectionPanel.cxx [hunk 14]
```diff
@@ -723,66 +755,70 @@ pqServer * PQSelectionPanel::getActiveServer()
 }
 
 //----------------------------------------------------------------------------
-void PQSelectionPanel::uuidKO(const std::string & uuid)
-{
+void PQSelectionPanel::uuidKO(const std::string & uuid) {
 	uuidItem[uuid]->setDisabled(true);
-	for (int idx_child = 0; idx_child < uuidItem[uuid]->childCount(); ++idx_child) {
+	for (int idx_child = 0; idx_child < uuidItem[uuid]->childCount();
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: unsigned? or size_t maybe?

File: src/PQSelectionPanel.cxx [hunk 15]
```diff
@@ -723,66 +755,70 @@ pqServer * PQSelectionPanel::getActiveServer()
...
-		uuidToPipeName[feuille.getUuid()]="EtpDocument";
-		if (feuille.getTimeIndex()<0 ) {
-			populateTreeView(feuille.getParent(), feuille.getParentType(), feuille.getUuid(), feuille.getName(), feuille.getType());
-		}
-		else  {
-			if(name_to_uuid.count(feuille.getName())<=0) {
-				populateTreeView(feuille.getParent(), feuille.getParentType(), feuille.getUuid(), feuille.getName(), feuille.getType());
-				name_to_uuid[feuille.getName()]=feuille.getUuid();
+		uuidToPipeName[feuille.getUuid()] = "EtpDocument";
+		if (feuille.getTimeIndex() < 0) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: no french! It should be "leaf"

File: src/PQSelectionPanel.cxx [hunk 16]
```diff
@@ -723,66 +755,70 @@ pqServer * PQSelectionPanel::getActiveServer()
...
 
-		PQToolsManager* manager = PQToolsManager::instance();
-		auto fesppReader = manager->getFesppReader();
+			PQToolsManager* manager = PQToolsManager::instance();
+			auto fesppReader = manager->getFesppReader();
 
-		if (fesppReader) {
-			pqActiveObjects *activeObjects = &pqActiveObjects::instance();
-			activeObjects->setActiveSource(fesppReader);
+			if (fesppReader) {
```
Comment (philippeVerney) [inline src/PQSelectionPanel.cxx]: nullptr

File: src/VTK/VtkGrid2DRepresentationCells.cxx [hunk 17]
```diff
@@ -55,10 +66,13 @@ VtkGrid2DRepresentationCells::~VtkGrid2DRepresentationCells()
...
 void VtkGrid2DRepresentationCells::createOutput(const std::string & uuid)
 {
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
 	if (!subRepresentation)	{
 
 		resqml2_0_1::Grid2dRepresentation* grid2dRepresentation = nullptr;
-		common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(uuid);
+		common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(uuid);
```
Comment (philippeVerney) [inline src/VTK/VtkGrid2DRepresentationCells.cxx]: you can use the template version of getDataObjectByUuid which automatically try to cast into Grid2dRepresentation.
You can then jsut check nullity at line below.

File: src/VTK/VtkGrid2DRepresentationPoints.cxx [hunk 18]
```diff
@@ -124,27 +143,43 @@ void VtkGrid2DRepresentationPoints::createOutput(const std::string & uuid)
...
-		common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid().substr(0, 36));
+//----------------------------------------------------------------------------
+long VtkGrid2DRepresentationPoints::getAttachmentPropertyCount(const std::string & uuid, const VtkEpcCommon::FesppAttachmentProperty propertyUnit)
+{
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
+	long result = 0;
+	resqml2_0_1::Grid2dRepresentation* grid2dRepresentation = nullptr;
+	common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid().substr(0, 36));
```
Comment (philippeVerney) [inline src/VTK/VtkGrid2DRepresentationPoints.cxx]: template version?

File: src/VTK/VtkGrid2DRepresentationPoints.cxx [hunk 19]
```diff
@@ -124,27 +143,43 @@ void VtkGrid2DRepresentationPoints::createOutput(const std::string & uuid)
...
-		long result = 0;
-		resqml2_0_1::Grid2dRepresentation* grid2dRepresentation = nullptr;
-		common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid().substr(0, 36));
+//----------------------------------------------------------------------------
+long VtkGrid2DRepresentationPoints::getAttachmentPropertyCount(const std::string & uuid, const VtkEpcCommon::FesppAttachmentProperty propertyUnit)
+{
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
+	long result = 0;
```
Comment (philippeVerney) [inline src/VTK/VtkGrid2DRepresentationPoints.cxx]: unsigned?

File: src/VTK/VtkPolylineRepresentation.cxx [hunk 20]
```diff
@@ -47,25 +47,46 @@ knowledge of the CeCILL license and that you accept its terms.
...
 //----------------------------------------------------------------------------
 void VtkPolylineRepresentation::createOutput(const std::string & uuid)
 {
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
 	if (!subRepresentation)	{
 		resqml2_0_1::PolylineSetRepresentation* polylineSetRepresentation = nullptr;
-		common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid());
+		common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid());
```
Comment (philippeVerney) [inline src/VTK/VtkPolylineRepresentation.cxx]: template?

File: src/VTK/VtkPolylineRepresentation.cxx [hunk 21]
```diff
@@ -120,25 +141,41 @@ void VtkPolylineRepresentation::createOutput(const std::string & uuid)
...
+//----------------------------------------------------------------------------
 long VtkPolylineRepresentation::getAttachmentPropertyCount(const std::string & uuid, const VtkEpcCommon::FesppAttachmentProperty propertyUnit)
 {
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
 	long result = 0;
 	resqml2_0_1::PolylineSetRepresentation* polylineSetRepresentation = nullptr;
-	common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid());
+	common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid());
```
Comment (philippeVerney) [inline src/VTK/VtkPolylineRepresentation.cxx]: template?

File: src/VTK/VtkSetPatch.cxx [hunk 22]
```diff
@@ -50,14 +50,24 @@ knowledge of the CeCILL license and that you accept its terms.
...
 VtkResqml2MultiBlockDataSet(fileName, name, uuid, uuidParent, idProc, maxProc), epcPackage(pck)
 {
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
 //	uuidIsChildOf[uuid] = new VtkEpcCommon();
 	resqml2_0_1::PolylineSetRepresentation* polylineSetRep = nullptr;
 	resqml2_0_1::TriangulatedSetRepresentation * triangulatedSetRep = nullptr;
-	common::AbstractObject *object = epcPackage->getResqmlAbstractObjectByUuid(uuid);
+	common::AbstractObject *object = epcPackage->getDataObjectByUuid(uuid);
```
Comment (philippeVerney) [inline src/VTK/VtkSetPatch.cxx]: template

File: src/VTK/VtkUnstructuredGridRepresentation.cxx [hunk 23]
```diff
@@ -50,27 +50,48 @@ knowledge of the CeCILL license and that you accept its terms.
...
 {
-
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
 	if (!subRepresentation)	{
 
 		resqml2_0_1::UnstructuredGridRepresentation* unstructuredGridRep = nullptr;
-		common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid());
+		common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid());
```
Comment (philippeVerney) [inline src/VTK/VtkUnstructuredGridRepresentation.cxx]: template?

File: src/VTK/VtkUnstructuredGridRepresentation.cxx [hunk 24]
```diff
@@ -197,26 +224,39 @@ vtkSmartPointer<vtkCellArray> VtkUnstructuredGridRepresentation::createOutputVtk
...
+//----------------------------------------------------------------------------
 long VtkUnstructuredGridRepresentation::getAttachmentPropertyCount(const std::string & uuid, const VtkEpcCommon::FesppAttachmentProperty propertyUnit)
 {
+#ifdef WITH_TEST
+	BEGIN_FUNC(__func__);
+#endif
 	long result = 0;
 	resqml2_0_1::UnstructuredGridRepresentation* unstructuredGridRep = nullptr;
-	common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid());
+	common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid());
```
Comment (philippeVerney) [inline src/VTK/VtkUnstructuredGridRepresentation.cxx]: template?

File: src/VTK/VtkWellboreTrajectoryRepresentationDatum.cxx [hunk 25]
```diff
@@ -55,7 +55,7 @@ void VtkWellboreTrajectoryRepresentationDatum::createOutput(const std::string &
 {
 	vtkOutput = vtkSmartPointer<vtkUnstructuredGrid>::New();
 	resqml2_0_1::WellboreTrajectoryRepresentation* wellboreSetRepresentation = nullptr;
-	common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(uuid);
+	common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(uuid);
```
Comment (philippeVerney) [inline src/VTK/VtkWellboreTrajectoryRepresentationDatum.cxx]: template?

File: src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx [hunk 26]
```diff
@@ -59,7 +59,7 @@ void VtkWellboreTrajectoryRepresentationPolyLine::createOutput(const std::string
 	if (!subRepresentation)	{
 
 		resqml2_0_1::WellboreTrajectoryRepresentation* wellboreSetRepresentation = nullptr;
-		common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid().substr(0, 36));
+		common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid().substr(0, 36));
```
Comment (philippeVerney) [inline src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx]: template?

File: src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx [hunk 27]
```diff
@@ -123,7 +123,7 @@ long VtkWellboreTrajectoryRepresentationPolyLine::getAttachmentPropertyCount(con
 {
 	long result = 0;
 	resqml2_0_1::WellboreTrajectoryRepresentation* wellboreSetRepresentation = nullptr;
-	common::AbstractObject* obj = epcPackageRepresentation->getResqmlAbstractObjectByUuid(getUuid().substr(0, 36));
+	common::AbstractObject* obj = epcPackageRepresentation->getDataObjectByUuid(getUuid().substr(0, 36));
```
Comment (philippeVerney) [inline src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx]: template?

File: src/VTK/log.h [hunk 28]
```diff
@@ -0,0 +1,191 @@
+#ifndef __LOG_H__
```
Comment (philippeVerney) [inline src/VTK/log.h]: license?

Comment (philippeVerney) [reply src/VTK/log.h]: Actually, this is the only remark which is a blocking one.

File: src/VTK/VtkUnstructuredGridRepresentation.cxx (hunk 24 above)
Comment (cedricRobert) [reply src/VTK/VtkUnstructuredGridRepresentation.cxx]: in a future version

File: src/VTK/VtkWellboreTrajectoryRepresentationDatum.cxx (hunk 25 above)
Comment (cedricRobert) [reply src/VTK/VtkWellboreTrajectoryRepresentationDatum.cxx]: in a future version

File: src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx (hunk 26 above)
Comment (cedricRobert) [reply src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx]: in a future version

File: src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx (hunk 27 above)
Comment (cedricRobert) [reply src/VTK/VtkWellboreTrajectoryRepresentationPolyLine.cxx]: in a future version

File: src/VTK/log.h (hunk 28 above)
Comment (cedricRobert) [reply src/VTK/log.h]: delete log.h

File: src/PQDataLoadManager.cxx [hunk 29]
```diff
@@ -111,22 +111,20 @@ void PQDataLoadManager::setupPipeline()
 		fesppReader = builder->createReader("sources", "Fespp", QStringList("EpcDocument"), this->Server);
 		manager->existPipe(true);
 	}
-	QStringList epcFiles =	this->ui->epcFile->filenames();//.join("*");
-
-//	QString epcFiles = this->ui->epcFile->filenames().join("*");
+	QStringList epcFiles =	this->ui->epcFile->filenames();
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: "this" could have been deleted

File: src/VTK/VtkEpcDocument.cxx [hunk 30]
```diff
@@ -1263,20 +1149,12 @@ void VtkEpcDocument::remove(const std::string & uuid)
...
 
 //----------------------------------------------------------------------------
 void VtkEpcDocument::attach()
 {
-#ifdef WITH_TEST
-	BEGIN_FUNC(__func__);
-#endif
-
-	for (unsigned int newBlockIndex = 0; newBlockIndex < attachUuids.size(); ++newBlockIndex)
+for (unsigned int newBlockIndex = 0; newBlockIndex < attachUuids.size(); ++newBlockIndex)
```
Comment (philippeVerney) [inline src/VTK/VtkEpcDocument.cxx]: size is a size_t datatype, not an unsigned int

File: src/VTK/VtkIjkGridRepresentation.cxx [hunk 31]
```diff
@@ -358,66 +280,42 @@ int VtkIjkGridRepresentation::getICellCount(const std::string & uuid) const
...
 	long result = 0;
 
-	if (subRepresentation)
-	{
+	if (subRepresentation)	{
 		// SubRep
 		result = jCellCount;
-	}else
-	{
+	}else	{
```
Comment (philippeVerney) [inline src/VTK/VtkIjkGridRepresentation.cxx]: }
else {

File: src/VTK/VtkIjkGridRepresentation.cxx [hunk 32]
```diff
@@ -358,66 +280,42 @@ int VtkIjkGridRepresentation::getICellCount(const std::string & uuid) const
...
 	long result = 0;
 
-	if (subRepresentation)
-	{
+	if (subRepresentation)	{
 		// SubRep
 		result = kCellCount;
-	}else
-	{
+	}else{
```
Comment (philippeVerney) [inline src/VTK/VtkIjkGridRepresentation.cxx]: }
else {

File: src/VTK/VtkWellboreTrajectoryRepresentation.cxx [hunk 33]
```diff
@@ -47,133 +47,78 @@ knowledge of the CeCILL license and that you accept its terms.
...
 {
-#ifdef WITH_TEST
-	BEGIN_FUNC(__func__);
-#endif
-		this->polyline.createOutput(uuid);
-		//	head.createOutput(uuid);
-#ifdef WITH_TEST
-	END_FUNC(__func__);
-#endif
+	this->polyline.createOutput(uuid);
```
Comment (philippeVerney) [inline src/VTK/VtkWellboreTrajectoryRepresentation.cxx]: "this" is unuseful

File: src/Fespp.cxx [hunk 34]
```diff
@@ -66,37 +66,38 @@ vtkCxxSetObjectMacro(Fespp, Controller, vtkMultiProcessController);
 
 //----------------------------------------------------------------------------
 Fespp::Fespp() :
-FileName(nullptr), Controller(nullptr), loadedFile(false), idProc(0), nbProc(0), countTest(0)
+		FileName(nullptr), Controller(nullptr), loadedFile(false), idProc(0), nbProc(0), countTest(0)
```
Comment (philippeVerney) [inline src/Fespp.cxx]: variable should start with lowercase

File: src/Fespp.cxx [hunk 35]
```diff
@@ -66,37 +66,38 @@ vtkCxxSetObjectMacro(Fespp, Controller, vtkMultiProcessController);
...
 	SetNumberOfInputPorts(0);
 	SetNumberOfOutputPorts(1);
 
 	loadedFile = false;
 
-	this->uuidList = vtkDataArraySelection::New();
+	uuidList = vtkDataArraySelection::New();
 
-	this->SetController(vtkMultiProcessController::GetGlobalController());
+	SetController(vtkMultiProcessController::GetGlobalController());
```
Comment (philippeVerney) [inline src/Fespp.cxx]: method should start lowercase

File: src/Fespp.cxx [hunk 36]
```diff
@@ -66,37 +66,38 @@ vtkCxxSetObjectMacro(Fespp, Controller, vtkMultiProcessController);
 
 //----------------------------------------------------------------------------
 Fespp::Fespp() :
-FileName(nullptr), Controller(nullptr), loadedFile(false), idProc(0), nbProc(0), countTest(0)
+		FileName(nullptr), Controller(nullptr), loadedFile(false), idProc(0), nbProc(0), countTest(0)
 {
 	SetNumberOfInputPorts(0);
```
Comment (philippeVerney) [inline src/Fespp.cxx]: method should start lowercase

File: src/Fespp.cxx [hunk 37]
```diff
@@ -66,37 +66,38 @@ vtkCxxSetObjectMacro(Fespp, Controller, vtkMultiProcessController);
 
 //----------------------------------------------------------------------------
 Fespp::Fespp() :
-FileName(nullptr), Controller(nullptr), loadedFile(false), idProc(0), nbProc(0), countTest(0)
+		FileName(nullptr), Controller(nullptr), loadedFile(false), idProc(0), nbProc(0), countTest(0)
 {
 	SetNumberOfInputPorts(0);
 	SetNumberOfOutputPorts(1);
```
Comment (philippeVerney) [inline src/Fespp.cxx]: method should start lowercase

File: src/Fespp.cxx [hunk 38]
```diff
@@ -135,15 +136,15 @@ void Fespp::SetSubFileName(const char* name)
 	if(isEpcDocument) {
 		if (std::find(fileNameSet.begin(), fileNameSet.end(),std::string(name))==fileNameSet.end())	{
 			fileNameSet.push_back(std::string(name));
-			this->OpenEpcDocument(name);
+			OpenEpcDocument(name);
```
Comment (philippeVerney) [inline src/Fespp.cxx]: variable should start lowercase

File: src/Fespp.cxx [hunk 39]
```diff
@@ -164,7 +165,7 @@ void Fespp::SetUuidList(const char* uuid, int status)
 		}
 #ifdef WITH_ETP
 		if(isEtpDocument && etpDocument!=nullptr) {
-				etpDocument->visualize(std::string(uuid));
+			etpDocument->visualize(std::string(uuid));
```
Comment (philippeVerney) [inline src/Fespp.cxx]: Do you really want to copy uuid?

File: src/Fespp.cxx [hunk 40]
```diff
@@ -179,23 +180,23 @@ void Fespp::SetUuidList(const char* uuid, int status)
 #endif
 	}
 
-	this->Modified();
-	this->Update();
-	this->UpdateDataObject();
-	this->UpdateInformation();
-	this->UpdateWholeExtent();
+	Modified();
```
Comment (philippeVerney) [inline src/Fespp.cxx]: method should start lowercase

File: src/Fespp.cxx [hunk 41]
```diff
@@ -179,23 +180,23 @@ void Fespp::SetUuidList(const char* uuid, int status)
...
+	UpdateDataObject();
+	UpdateInformation();
+	UpdateWholeExtent();
 }
 
 //----------------------------------------------------------------------------
 int Fespp::GetNumberOfuuidListArrays()
 {
-	return this->uuidList->GetNumberOfArrays();
+	return uuidList->GetNumberOfArrays();
```
Comment (philippeVerney) [inline src/Fespp.cxx]: method should start lowercase

File: src/Fespp.cxx [hunk 42]
```diff
@@ -179,23 +180,23 @@ void Fespp::SetUuidList(const char* uuid, int status)
...
 }
 
 //----------------------------------------------------------------------------
 const char* Fespp::GetuuidListArrayName(int index)
 {
-	return this->uuidList->GetArrayName(index);
+	return uuidList->GetArrayName(index);
```
Comment (philippeVerney) [inline src/Fespp.cxx]: method should start lowercase

File: src/PQDataLoadManager.cxx [hunk 43]
```diff
@@ -63,32 +63,32 @@ PQDataLoadManager::PQDataLoadManager(QWidget* p, Qt::WindowFlags f /*=0*/)
 : QDialog(p, f)
 {
 	PQToolsManager* manager = PQToolsManager::instance();
-	this->Server = manager->getActiveServer();
+	Server = manager->getActiveServer();
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: variable should start lowercase

File: src/PQDataLoadManager.cxx [hunk 44]
```diff
@@ -63,32 +63,32 @@ PQDataLoadManager::PQDataLoadManager(QWidget* p, Qt::WindowFlags f /*=0*/)
...
 	PQToolsManager* manager = PQToolsManager::instance();
-	this->Server = manager->getActiveServer();
+	Server = manager->getActiveServer();
 
 	this->ui = new PQDataLoadManager::pqUI;
 	this->ui->setupUi(this);
 
-	this->ui->epcFile->setServer(this->Server);
-	this->ui->epcFile->setForceSingleFile(true);
+	this->ui->epcFile->setServer(Server);
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: "this" is unuseful

File: src/PQDataLoadManager.cxx [hunk 45]
```diff
@@ -63,32 +63,32 @@ PQDataLoadManager::PQDataLoadManager(QWidget* p, Qt::WindowFlags f /*=0*/)
...
 {
-	delete this->ui;
+	delete ui;
 }
 
 //-----------------------------------------------------------------------------
 void PQDataLoadManager::checkInputValid()
 {
-	auto valid = (this->ui->epcFile->filenames().isEmpty()) ? false : true;
+	auto valid = (ui->epcFile->filenames().isEmpty()) ? false : true;
```
Comment (philippeVerney) [inline src/PQDataLoadManager.cxx]: bool valid = !ui->epcFile->filenames().isEmpty()

File: src/PQEtpConnectionManager.cxx [hunk 46]
```diff
@@ -86,15 +86,15 @@ void PQEtpConnectionManager::setupPipeline()
...
+		fesppReader = builder->createReader("sources", "Fespp", QStringList("EtpDocument"), Server);
 		manager->etp_existPipe(true);
 	}
 
 	if (fesppReader != nullptr) {
 		pqActiveObjects *activeObjects = &pqActiveObjects::instance();
 		activeObjects->setActiveSource(fesppReader);
 
-		auto connection_parameter = this->ui->etp_ip->text()+":"+this->ui->etp_port->text();
+		auto connection_parameter = this->ui->etp_ip->text()+":"+ this->ui->etp_port->text();
```
Comment (philippeVerney) [inline src/PQEtpConnectionManager.cxx]: "this" is unuseful

File: src/PQEtpPanel.cxx [hunk 47]
```diff
@@ -32,24 +32,24 @@ void PQEtpPanel::constructor()
 	ui.setupUi(t_widget);
 	setWidget(t_widget);
 
-	EtpStatus_Button = ui.status;
+	this->etpStatus_Button = ui.status;
```
Comment (philippeVerney) [inline src/PQEtpPanel.cxx]: "this" is unuseful

File: src/PQToolsActionGroup.cxx [hunk 48]
```diff
@@ -40,17 +40,16 @@ PQToolsActionGroup::PQToolsActionGroup(QObject* p)
 : QActionGroup(p)
 {
 	PQToolsManager* manager = PQToolsManager::instance();
-	if (!manager)
-	{
+	if (!manager) {
```
Comment (philippeVerney) [inline src/PQToolsActionGroup.cxx]: nullptr check

File: src/PQToolsManager.cxx [hunk 49]
```diff
@@ -299,26 +289,22 @@ pqPipelineSource* PQToolsManager::findPipelineSource(const char* SMName)
 //-----------------------------------------------------------------------------
 pqView* PQToolsManager::findView(pqPipelineSource* source, int port, const QString& viewType)
 {
-	if (source)
-	{
+	if (source) {
```
Comment (philippeVerney) [inline src/PQToolsManager.cxx]: nullptr check

File: src/PQToolsManager.cxx [hunk 50]
```diff
@@ -330,65 +316,63 @@ pqView* PQToolsManager::findView(pqPipelineSource* source, int port, const QStri
...
 //----------------------------------------------------------------------------
 void PQToolsManager::deletePipelineSource(pqPipelineSource* pipe)
 {
-	if(this->existPipe())
+	if(existPipe())
 	{
 		pqPipelineSource * source = findPipelineSource("EpcDocument");
-		if (!source)
-		{
+		if (!source) {
```
Comment (philippeVerney) [inline src/PQToolsManager.cxx]: nullptr check

File: src/PQToolsManager.cxx [hunk 51]
```diff
@@ -330,65 +316,63 @@ pqView* PQToolsManager::findView(pqPipelineSource* source, int port, const QStri
...
 			getPQSelectionPanel()->deleteTreeView();
-			this->existEpcPipe = false;
+			existEpcPipe = false;
 		}
 #ifdef WITH_ETP
 		source = findPipelineSource("EtpDocument");
-		if (!source)
-		{
+		if (!source) {
```
Comment (philippeVerney) [inline src/PQToolsManager.cxx]: nullptr check

File: src/VTK/VtkGrid2DRepresentationCells.cxx [hunk 52]
```diff
@@ -84,23 +83,19 @@ void VtkGrid2DRepresentationCells::createOutput(const std::string & uuid)
...
-			for (unsigned int j = 0; j < nbNodeJ; ++j)
-			{
-				for (unsigned int i = 0; i < nbNodeI; ++i)
-				{
+			for (unsigned int j = 0; j < nbNodeJ; ++j) {
+				for (unsigned int i = 0; i < nbNodeI; ++i) {
 					unsigned int ptId = i + j * nbNodeI;
-					if (vtkMath::IsNan(z[ptId]))
-					{
+					if (vtkMath::IsNan(z[ptId])) {
```
Comment (philippeVerney) [inline src/VTK/VtkGrid2DRepresentationCells.cxx]: I would rather use the standard isNaN function than the vtk one : https://en.cppreference.com/w/cpp/numeric/math/isnan

**Pull request merged**